### PR TITLE
Add screenshot button

### LINKED ISSUE #55: add ability to share tasks as image
add a button that, when clicked, copies an image of the tasks to the clipboard.

i've been studying how to get this done, these links may be useful.

https://www.npmjs.com/package/html-to-image#toPng
https://chat.openai.com/share/2f2b6625-e84f-469e-8e4c-42e6ac0ea449

example: i have these 5 tasks. i click a button. and the image below is in my clipboard to be pasted anywhere i want.

image

### PR DESCRIPTION
**Pull request by brenodanyel**
This PR resolves #55. 

Let me know if you have any suggestions.

### CODE REVIEW

**Review by LukeberryPi — COMMENTED** [review]
quick 2 comments

we may update this button's layout/position, IMO it shouldn't have the same proeminance as other header options. i'm currently thinking of some relative position to the tasks itself, and maybe a button with different text or maybe no text. more on this soon.

i'm requesting the changes mentioned as comments, talk soon,

File: src/components/Tasks.tsx [hunk 1]
```diff
@@ -89,7 +89,7 @@ export function Tasks() {
                 onChange={(event) => handleChange(event, idx)}
                 autoFocus={isFirstTask}
                 autoComplete="off"
-                placeholder={`${isFirstTask ? `${placeholder}?` : `task-${idx + 1}`}`}
```
[the remark this answers is not in the document]
Comment (brenodanyel) [reply src/components/Tasks.tsx]: I saw that you also removed it from the main upstream branch, so I won't touch.

File: src/icons/ScreenshotIcon.tsx
[the remark this answers is not in the document]
Comment (brenodanyel) [reply src/icons/ScreenshotIcon.tsx:1]: Changed, thank you!

Maybe it's worth adding this information somewhere? Like contributing.md or even readme.md

**Comment by LukeberryPi** [comment]
as mentioned [in the issue](https://github.com/LukeberryPi/phived/issues/55), we will need to adapt the generated image as shown below (with some padding and simulated hover effect on the first task). if we are unable to do so, the feature will not be released for now.

furthermore, we will preserve only the functionality for copying to clipboard. even though the png download is an interesting idea, it strays from the original concept. with this in mind, we will switch to an alert window, with the text "an image of your tasks have been copied to the clipboard!"

i'll dive into the code if needed be, but just thought i'd share some thoughts on this pull request for us to move forward.

image

**Review by LukeberryPi — COMMENTED** [review]
changes look fine, the main problem is that the generated image isn't on par with the expected behavior. we need padding and hover effect on the first task. furthermore, the only feature to be added is copy to clipboard, no png will be downloadable.

**Comment by LukeberryPi** [comment]
closing due to inactivity.

**Pull request closed without merging**